spec/ios_send_app_size_metrics_spec.rb

```diff
@@ -0,0 +1,102 @@
+require_relative './spec_helper'
+
+describe Fastlane::Actions::IosSendAppSizeMetricsAction do
+  let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'app_size_metrics') }
+  let(:fake_ipa_size) { 1337 } # The value used in the `app-thinning.plist` and `ios-metrics-payload.json` fixtures
```

Comment:
Is this an easter egg reference to the [Leet language](https://en.wikipedia.org/wiki/Leet)?